digdag-standards/src/test/java/io/digdag/standards/operator/td/TdOperatorFactoryTest.java

```diff
@@ -93,9 +110,10 @@ public void testTDJobRequestParamsWithEngineVersion()
         when(op.getDatabase()).thenReturn("testdb");
 
         TDJobRequest jobRequest = testTDJobRequestParams(projectPath, config);
+        String sql = createStmtComment(taskRequest).append("select 1").toString();
```

Comment:
There is no test to check `createStmtComment()` works expectedly?

Reply:
Added a test case for `wrapStmtWithComment()`  e55103b